packages/ag-grid-community/src/entities/gridOptions.ts

```diff
@@ -690,18 +690,15 @@ export interface GridOptions<TData = any> {
      */
     advancedFilterBuilderParams?: IAdvancedFilterBuilderParams;
     /**
-     * By default, Advanced Filter sanitises user input and passes it to `new Function()` to provide the best performance.
-     * Set to `true` to prevent this and use defined functions instead.
-     * This will result in slower filtering, but it enables Advanced Filter to work when `unsafe-eval` is disabled.
-     * @default false
+     * @deprecated As of v34, advanced filter evaluation is deprecated, and this option will default to true.
```

Comment:
```suggestion
     * @deprecated As of v34, advanced filter no longer uses function evaluation, so this option has no effect.
```

Perhaps something like this instead to clarify that it does nothing